### PR TITLE
Save the last evalutate expression to $_ (as google chrome does)

### PR DESCRIPTION
**Pull request by potyl**
Add support for [$_ as Google Chrome](https://developer.chrome.com/devtools/docs/commandline-api#_) does.

### CODE REVIEW

File: stetho-js-rhino/src/main/java/com/facebook/stetho/rhino/JsRuntimeReplFactoryBuilder.java [hunk 1]
```diff
@@ -221,7 +226,13 @@ private void importVariables(@NonNull ScriptableObject scope) throws StethoJsExc
       String varName = entrySet.getKey();
       Object varValue = entrySet.getValue();
       try {
-        Object jsValue = Context.javaToJS(varValue, scope);
+        Object jsValue;
+        if (varValue instanceof Scriptable || varValue instanceof Undefined) {
+          jsValue = varValue;
+        }
+        else {
```
Comment (jasta) [inline stetho-js-rhino/src/main/java/com/facebook/stetho/rhino/JsRuntimeReplFactoryBuilder.java]: style nit: else on same line as `}`

**Comment by jasta** [comment]
Looks good except the minor style nit.

**Comment by potyl** [comment]
Ill fix it

File: stetho-js-rhino/src/main/java/com/facebook/stetho/rhino/JsRuntimeReplFactoryBuilder.java (hunk 1 above)
Comment (potyl) [inline stetho-js-rhino/src/main/java/com/facebook/stetho/rhino/JsRuntimeReplFactoryBuilder.java]: fixed

**Pull request merged**